packages/aragon-wrapper/src/index.js

```diff
@@ -772,6 +774,27 @@ export default class Aragon {
     return this.accounts.pipe(first()).toPromise()
   }
 
+  /**
+   * Allows apps to sign arbitrary data via a RPC call
+   *
+   * @param {Array<Object>} params An object containing the address of the app and the message to be signed
+   * @return {Promise<string>} signature hash
+   */
+  signMessage (message, requestingApp) {
+    return new Promise((resolve, reject) => {
+      this.signatures.next({
+        requestingApp,
+        message,
+        accept (signatureHash) {
```

Comment:
@sohkai I thought the long term goal was to rename to `resolve` as we did with [identity](https://github.com/aragon/aragon.js/blob/31701f5e183ce8e5dca4785da1d2843bb75f5297/packages/aragon-wrapper/src/index.js#L519) This is not critical for this PR but should be consistent. 

Any preferences?

Reply:
Ah yes, good catch. Let's rename this to `resolve()` 👍.